components/rhel/rhel-ose/Vagrantfile

```diff
@@ -47,10 +47,8 @@ Vagrant.configure(2) do |config|
     config.registration.password = ENV['SUB_PASSWORD']
   end
 
-  config.vm.provision "shell", inline: <<-SHELL
-    systemctl enable openshift
-    systemctl start openshift
-  SHELL
+  # vagrant-service-manager plugin will configure openshift service by default for CDK
+  # config.servicemanager.services = "docker, openshift"
```

Comment:
> However if we are going with the change , it wont be backward compatible with service-manager < 1.0.0?

0.0.5 version plugin had support to configure openshift service by default, however there was a bug <https://github.com/projectatomic/vagrant-service-manager/issues/131> , which is fixed in 1.0.0 release of the plugin. 
So 1.0.0 version of the plugin is required.